commands/user.go

```diff
@@ -315,3 +329,45 @@ auth_service: {{.AuthService}}`
 
 	return nil
 }
+
+func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error {
+	// TODO What to do with err?
+	page, errp := command.Flags().GetInt("page")
+	if errp != nil {
+		return errors.Wrap(errp, "Could not parse --page")
+	}
+	perPage, errpp := command.Flags().GetInt("per-page")
```

Comment:
I think we can reuse the same error variable. There's no need to name a new one.